bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -45,197 +41,74 @@
+  </p>
 </section>
 
-<section>
-  <h3 data-accordion-role="tab">{{ _('What data is being collected?') }}</h3>
-  <div data-accordion-role="tabpanel">
-    <p>
-      {% if LANG in ['en-US'] %}
-        Mozilla collects Tiles related data such as number of clicks, impressions and Tile specific
-        data (e.g. position and size of grid) to help Mozilla determine how frequently the Tile has
-        been seen or interacted with, as well as your IP address (collected by Firefox, quickly
-        translated into a region code and then deleted).
-      {% else %}
-      {% trans %}
-        Aggregate data from the New Tab page includes impressions, clicks and tile placement.
-      {% endtrans %}
-      {% endif %}
-    </p>
-  </div>
+<section class="tls-discover">
+  <h2>{{ _('Discover new sites and content on the Web') }}</h2>
+  <p>{{ _('If enabled, Firefox displays website links as thumbnails (miniaturized images) or logos on your New Tab page. These are called Tiles.') }}</p>
+  <p>
+    {{ _('Tiles save you time by giving you quick access to your frequently visited sites, as well as showing new sites and content that are similar to sites you already love.') }} 
+    {{ _('Here are the types of Tiles you may see:') }}
+  </p>
+
+  <h4>{{ _('Top sites') }}</h4>
+  <p>{{ _('New Tab will show the “top sites” you visited most recently and frequently, so you can access them quickly.') }}</p>
+  <img src="{{ static('img/firefox/tiles/top-sites.png') }}" alt="{{ _('Screenshot') }}">
+
+  <h4>{{ _('Suggested sites') }}</h4>
+  <p>
+    {{ _('See sites and content suggested to you based on categories that Firefox identifies in your browsing history.') }} 
+    {{ _('They will be clearly labeled in your new tab as “Suggested,” or when the suggestion is from a commercial partner, they will be labeled as “Sponsored,” and will indicate the identified interest category.') }}
+  </p>
+  <img src="{{ static('img/firefox/tiles/suggested-sites.png') }}" alt="{{ _('Screenshot') }}">
 </section>
 
-<section>
-  <h3 data-accordion-role="tab">{{ _('What data is collected when I opt out?') }}</h3>
-  <div data-accordion-role="tabpanel">
-    <p>
-      {% if LANG in ['en-US'] %}
-        No data is collected when a user deactivates the Enhanced Tiles experience.
-      {% else %}
-      {% trans %}
-        No data is collected when a user chooses to opt out of the enhanced Tiles experience.
-      {% endtrans %}
-      {% endif %}
-    </p>
-  </div>
+<section class="tls-started">
+  <h2>{{ _('If you’re getting started') }}</h2>
+  <p>
+    {{ _('If you’re using Firefox for the first time, the New Tab page is already populated with useful and interesting sites and content.') }} 
+    {{ _('This includes both referrals to sponsored sites from commercial partners, labeled as "Sponsored," and content we consider important to our mission of advancing the public benefit of the Internet.') }}
```

Comment:
Could we have “smart quotes” here for consistency please? I'd prefer we make the change now rather than breaking the string freeze for that minor change.